doc/source/style.ipynb

```diff
@@ -1216,7 +1166,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [default]",
+   "display_name": "Python 3",
    "language": "python",
    "name": "python3"
```

Comment:
When building the docs locally, I always have to rename the kernel to `pandas-dev` here. Not necessarily relevant for this PR, but it would be great if that could be modularized.